src/OrbitBase/include/OrbitBase/ThreadUtils.h

```diff
@@ -16,17 +16,69 @@
 
 namespace orbit_base {
 
-#ifdef _WIN32
+static constexpr uint32_t kInvalidThreadId = 0xffffffff;
```

Comment:
Why are those moved from ThreadConstants? While it is fine for me to have them here, it feels a bit strange that now we have two places for thread constants.

Reply:
Moved back to ThreadConstants.h and include that header in ThreadUtils.h.